recipe/build.sh

```diff
@@ -1,43 +1,60 @@
 #!/bin/bash
 
+./configure \
+  --prefix=$PREFIX \
+  --build=$BUILD \
+  --host=$HOST \
+  --without-debug \
+  --without-ada \
+  --without-manpages \
+  --with-shared \
+  --disable-overwrite \
+  --enable-symlinks \
+  --enable-termcap \
+  --with-pkg-config-libdir=${PREFIX}/lib/pkgconfig \
+  --enable-pc-files \
+  --with-termlib \
+  --enable-widec
+make -j${CPU_COUNT} ${VERBOSE_AT}
+make install
 
-for USE_WIDEC in false true;
-do
-    WIDEC_OPT=""
-    if [ "${USE_WIDEC}" = true ];
-    then
-        WIDEC_OPT="--enable-widec"
-    fi
-
-    sh ./configure \
-	    --prefix=$PREFIX \
-	    --without-debug \
-	    --without-ada \
-	    --without-manpages \
-	    --with-shared \
-	    --with-pkg-config \
-	    --disable-overwrite \
-	    --enable-symlinks \
-	    --enable-termcap \
-	    --enable-pc-files \
-	    --with-termlib \
-	    $WIDEC_OPT
-    make
-    make install
-    make clean
-    make distclean
+if [[ $(uname -s) == Linux ]]; then
+  _SOEXT=.so
+else
+  _SOEXT=.dylib
+fi
```

Comment:
This is already done by `conda-build` with `SHLIB_EXT`. So can just use `SHLIB_EXT` instead of `_SOEXT` below as needed.